addons/binding/org.openhab.binding.onebusaway/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: OneBusAway Binding
+Bundle-SymbolicName: org.openhab.binding.onebusaway;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
```

Comment:
Fairly certain this was generated; probably want to check that the generation script is updated.